code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -528,22 +529,38 @@ var/list/ai_verbs_default = list(
 	if(check_unable())
 		return
 
-	var/input
+	var/received //var/input caused too many problems, because there's also the input proc.
 	var/choice = alert("Would you like to select a hologram based on a (visible) crew member, switch to unique avatar, or load your character from your character slot?",,"Crew Member","Unique","My Character")
 
 	switch(choice)
 		if("Crew Member") //A seeable crew member (or a dog)
-			var/list/targets = trackable_mobs()
+			var/list/targets = list()
+			for(var/mob/living/M in mob_list)
+				if(!M)
+					continue
+				if(!M.client) //This *will* keep SSD players from showing up on the list, but if this wasn't here, there would be a runtime if you tried to imitate an SSD mob.
+					continue
+				if(M == usr)
+					continue
+				if(M.tracking_status() != TRACKING_POSSIBLE)
+					continue
+
+				targets += M
+
 			if(targets.len)
-				input = input("Select a crew member:") as null|anything in targets //The definition of "crew member" is a little loose...
-				//This is torture, I know. If someone knows a better way...
-				if(!input) return
-				var/new_holo = getHologramIcon(getCompoundIcon(targets[input]))
+				received = input("Select a crew member to imitate:") as null|anything in targets
+				var/mob/living/carbon/human/dummy/dummy = received
+				dummy.client.prefs.dress_preview_mob(dummy)

```

Comment:
The dummy will not have a client. You need to call `target.client.prefs.dress_preview_mob(dummy)`, so you are asking the real mob's client to dress this dummy preview mob you've made in nullspace.